docs/iciclecharts-with-cpu-profiles.mdx

```diff
@@ -0,0 +1,31 @@
+# Iciclecharts with CPU profiling data
+
+Iciclechart is a visualization method that allows you to see the CPU usage of your application over time. In other words, it is similar to a flamegraph, but has an additonal time dimension. This means that you can see how the CPU usage of your application changes over time, and how it is affected by different events.
+
+Screenshot of Iciclechart with CPU profiling data:
+![Iciclechart with CPU profiling data](/img/vizualisation/iciclechart-example.png)
+
+## Prerequisite
+Before you can use Iciclechart with CPU profiling data, you need to be collecting the CPU profiling data with `cpu` or `thread_id` labels. You can do with by adding the following section to the parca agent configuration if you don't have it already:
+```yaml
+  relabel_configs:
+    - source_labels: [__meta_thread_id]
+      target_label: thread_id
+    - source_labels:
+      - __meta_cpu
```

Comment:
let's use the same formatting for both meta labels

Reply:
Done